canvas/base_test.go

```diff
@@ -0,0 +1,55 @@
+package canvas
+
+import (
+	"testing"
+
+	"fyne.io/fyne"
+	"github.com/stretchr/testify/assert"
+)
+
+// These tests group a few methods by function
+
+func TestBase_Resize(t *testing.T) {
+	b := &baseObject{}
+	targetSize := fyne.NewSize(50, 50)
+	b.Resize(targetSize)
+
+	assert.Equal(t, b.Size(), targetSize)
+}
+
+func TestBase_Move(t *testing.T) {
+	originPosition := fyne.NewPos(0, 0)
```

Comment:
Might be worth testing with a non-default starting point - if something goes wrong and the code defaults to 0,0 it won't show up here.

Reply:
I agree, I added  a  second test that checks just  this.